pkg/machine/config.go

```diff
@@ -400,3 +400,13 @@ func (v VMType) String() string {
 	}
 	return "qemu"
 }
+
+type APIForwardingState int
+
+const (
+	NoForwarding APIForwardingState = iota
+	ClaimUnsupported
+	NotInstalled
+	MachineLocal
+	DockerGlobal
```

Comment:
I know you just moved this but considering that they are no public could add comments what these mean. I have no idea what the differences between these are.

Reply:
i actually dont know either.